homework_04/12_Martin_Vrachev/test_circles.py

```diff
@@ -0,0 +1,118 @@
+'''
+Unit tests for circles.py
+'''
+
+import pytest
+from circles import distance, Point, Circle, RelativePosition
+
+def test_create_point():
+    '''
+    Tests for point creation
+    '''
+    point_1 = Point(5.4, 6.5)
+    assert isinstance(point_1, Point)
+
+def test_create_invalid_point():
+    '''
+    Tests for creating invalid point
+    '''
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        Point('koko', 5.7)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_create_circle():
+    '''
+    Tests circle creation
+    '''
+    point_1 = Point(5.5, 4.5)
+    circle_1 = Circle(point_1, 10.0)
+    assert isinstance(circle_1, Circle)
+
+def test_create_invalid_circle():
+    '''
+    Tests for creating invalid circles
+    '''
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        Circle(5, 6.7)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        point_1 = Point(1.2, 8.7)
+        Circle(point_1, 'radius')
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_distance():
+    '''
+    Tests for distance between two points
+    '''
+    point_1 = Point(2.0, 1.0)
+    point_2 = Point(6.0, 4.0)
+    assert distance(point_1, point_2) == 5.0
+
+    point_1 = Point(2.0, -1.0)
+    point_2 = Point(-2.0, 2.0)
+    assert distance(point_1, point_2) == 5.0
+
+def test_invalid_distance_arguments():
+    '''
+    Tests for invalid araguments
+    '''
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        distance(2,3)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_find_realtive_position_invalid_arguments():
```

Comment:
*relative
/# pylint: disable=invalid-name will help you as I do not see anything wrong with the name.

Reply:
Yeah, I will fix the typo, thanks. However I do not get an error from pylint here as well.